habitat/core/env.py

```diff
@@ -366,7 +366,7 @@ def step(self, action: int) -> Tuple[Observations, Any, bool, dict]:
 
         return observations, reward, done, info
 
-    def seed(self, seed: int) -> None:
+    def seed(self, seed: Optional[int] = None) -> None:
```

Comment:
I did this to keep signature consistency with [gym.Env](https://github.com/openai/gym/blob/9dea81b48a2e1d8f7e7a81211c0f09f627ee61a9/gym/core.py#L115), since RLEnv is a subclass of gym.Env.